.gitignore

```diff
@@ -18,3 +18,4 @@ logs.txt
 packages/ethereum/types/
 packages/ethereum/lib/types/
 packages/utils/lib/types/
+types/
```

Comment:
Please be more specific! This confuses VSCode

Reply:
e.g. `packages/*/types`

Reply:
Done.